### PR TITLE
Add top 10 words

### PR DESCRIPTION
**Pull request by vanishedvan**

### CODE REVIEW

File: src/app/extractor.js [hunk 1]
```diff
@@ -123,7 +123,7 @@ export const extractData = async (zip) => {
     console.log(`[debug] ${extractedData.channels.length} channels loaded.`);
 
     const words = extractedData.channels.map((channel) => channel.messages).flat().map((message) => message.words).flat().filter((w) => w.length > 5);
-    extractedData.favoriteWord = mostOccurences(words);
+    extractedData.favoriteWord = mostOccurences(words, true).filter((word) => !['!', '?', '/', '<', '@'].includes(word[0])).slice(0, 10).join(', ');
```
Comment (jack1142) [inline src/app/extractor.js:126]: I think that part of the fun is that the words starting with these characters are *not* excluded and excluding words starting with these characters does hurt the quality of the returned statistics.

File: src/views/Home.svelte [hunk 2]
```diff
@@ -53,7 +53,7 @@ const hoursLabels = new Array(24).fill(0).map((v, i) => i == 0 ? '12am' : i < 12
             </div>
             <div class="fun-fact">
                 <svg fill="none" stroke="currentColor" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M4.318 6.318a4.5 4.5 0 000 6.364L12 20.364l7.682-7.682a4.5 4.5 0 00-6.364-6.364L12 7.636l-1.318-1.318a4.5 4.5 0 00-6.364 0z"></path></svg>
-                <h3 style="margin-left: 10px;">Your favorite word is <span class="text-discord">{ $data.favoriteWord }</span></h3>
+                <h3 style="margin-left: 10px;">Your favorite words are <span class="text-discord">{ $data.favoriteWord }.</span></h3>
```
Comment (jack1142) [inline src/views/Home.svelte:56]: It does not appear that you include counts of the words here which was part of the request in #4.
I guess it could be a separate PR though, but that's not my call 😄

File: src/app/extractor.js (hunk 1 above)
Comment (Androz2091) [reply src/app/extractor.js:126]: Yes, if we keep the check line 125 (to keep only words longer than 5) I think that we can remove these filters. The point is just to remove the non-words such as `I`, `?`, or `.`.

Comment (Androz2091) [inline src/app/extractor.js:126]: I think that it should be an array `favoriteWords`. The `join()` should be done on the main page with Svelte

File: src/views/Home.svelte (hunk 2 above)
Comment (Androz2091) [reply src/views/Home.svelte:56]: Maybe we could add tooltips so when you hover the word you have the number of occurences?

File: src/app/extractor.js (hunk 1 above)
Comment (vanishedvan) [reply src/app/extractor.js:126]: That is a better solution.

File: src/views/Home.svelte (hunk 2 above)
Comment (vanishedvan) [reply src/views/Home.svelte:56]: I didn't plan on adding the counts immediately, I will most likely add them soon. Tooltips sound like a better idea than showing the count as text.

File: src/app/extractor.js (hunk 1 above)
Comment (vanishedvan) [reply src/app/extractor.js:126]: Ok, I will change that.

**Comment by vanishedvan** [comment]
@Androz2091 @jack1142 I have made the changes you requested.

**Review by Androz2091 — APPROVED** [review]
Thank you for your contribution 👍

**Comment by vanishedvan** [comment]
No problem!
